sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryClient.java

```diff
@@ -0,0 +1,85 @@
+     * Unless configured while building this client through {@link ModelsRepositoryClientBuilder#serviceVersion(ModelsRepositoryServiceVersion)},
+     * this value will be equal to the latest service API version supported by this client.
+     *
+     * @return The Azure Models Repository service API version.
+     */
+    public ModelsRepositoryServiceVersion getServiceVersion() {
+        return this.modelsRepositoryAsyncClient.getServiceVersion();
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Map<String, String> getModels(String dtmi) {
+        return modelsRepositoryAsyncClient.getModels(dtmi).block();
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi                 A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @param dependencyResolution A {@link ModelsDependencyResolution} value to force model resolution behavior.
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Map<String, String> getModels(String dtmi, ModelsDependencyResolution dependencyResolution, Context context) {
+        return modelsRepositoryAsyncClient.getModels(dtmi, dependencyResolution, context).block();
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmis Collection of well-formed DTDL model Ids
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Map<String, String> getModels(Iterable<String> dtmis) {
+        return modelsRepositoryAsyncClient.getModels(dtmis).block();
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmis                An Iterable of well-formed DTDL model Ids. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @param dependencyResolution A {@link ModelsDependencyResolution} value to force model resolution behavior.
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
```

Comment:
Is adding code snippet a todo for later?

Reply:
Yes, once we add samples.